commit 83ee0681993dfe68868683bdc02042ec025ac38f
Author: LI Qingwu <Qing-wu.Li@leica-geosystems.com.cn>
Date:   Fri Mar 19 11:02:35 2021 +0000

    dt-bindings: power: bq27xxx: add bq78z100
    
    Add bindings for TI BQ78Z100. An I2C interface gas gauge.
    It provides a fully integrated safety protection
    and authentication for 1 to 2-series cell Li-Ion and
    Li-Polymer battery packs.
    
    Signed-off-by: LI Qingwu <Qing-wu.Li@leica-geosystems.com.cn>
    Acked-by: Rob Herring <robh@kernel.org>
    Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>

diff --git a/Documentation/devicetree/bindings/power/supply/bq27xxx.yaml b/Documentation/devicetree/bindings/power/supply/bq27xxx.yaml
index 45beefccf31a..712e974b28b6 100644
--- a/Documentation/devicetree/bindings/power/supply/bq27xxx.yaml
+++ b/Documentation/devicetree/bindings/power/supply/bq27xxx.yaml
@@ -52,6 +52,7 @@ properties:
       - ti,bq27z561
       - ti,bq28z610
       - ti,bq34z100
+      - ti,bq78z100
 
   reg:
     maxItems: 1